pilot/pkg/serviceregistry/serviceentry/servicediscovery_test.go

```diff
@@ -1485,7 +1485,7 @@ func Test_autoAllocateIP_conditions(t *testing.T) {
 					Resolution:               model.ClientSideLB,
 					DefaultAddress:           "0.0.0.0",
 					AutoAllocatedIPv4Address: "240.240.0.1",
-					AutoAllocatedIPv6Address: "2001:0002::f0f0:01",
+					AutoAllocatedIPv6Address: "2001:2::f0f0:01",
```

Comment:
Since you are changing 0002 to 2, it probably will be quite nice to also change the 01 at the end to 1?

Reply:
thanks for the kindly remind!